Source/AtomicApp/IPCServerApp.cpp

```diff
@@ -0,0 +1,261 @@
+#include <Atomic/IO/Log.h>
+#include <Atomic/IO/FileSystem.h>
+
+#include <Atomic/IO/IOEvents.h>
+#include <Atomic/Core/ProcessUtils.h>
+#include <Atomic/Core/CoreEvents.h>
+
+#include <Atomic/IPC/IPC.h>
+#include <Atomic/IPC/IPCEvents.h>
+#include <Atomic/IPC/IPCWorker.h>
+#include <Atomic/IPC/IPCBroker.h>
+
+#include <ToolCore/ToolSystem.h>
```

Comment:
Need to be very careful dragging dependencies across modules, this is introducing ToolCore into AtomicApp, which we really cannot do... see review summary comment